uint8 pin
float32 us
---
uint8 result
